CONTRIBUTING.rst

```diff
@@ -134,57 +134,55 @@ Documentation
 C++
 ---
 
-.clang-format
-~~~~~~~~~~~~~
-
-* When starting a new C++ project, copy the .clang-format file included here.
-* Our style is based on Google (as opposed to LLVM, Chromium, Mozilla, or Webkit) with minor differences.
-* :code:`ColumnLimit` is set to :code:`120`, as specified in `Coding Conventions`_.
-* :code:`NamespaceIndentation` is set to :code:`Inner` as a middle ground between :code:`None` (Google) and :code:`All`,
-  such that every line in a file defining a namespace isn't indented, but nested namespaces are easily spotted.
-* Various indent-width specifiers are scaled by a factor of 2 such that the base indent is :code:`4`, as specified in
-  `Coding Conventions`_, instead of :code:`2` (Google). This is especially helpful for readibility in cases like
-
-  .. code-block:: c++
-
-    if (condition) {
-        foo();
-    } else {
-        bar();
-    }
-
-  as opposed to
+C++ Version
+~~~~~~~~~~~
 
-  .. code-block:: c++
+C++ code should be compatible with standard C++11.
 
-    if (condition) {
-      foo();
-    } else {
-      bar();
-    }
+Naming
+~~~~~~
 
-Additional Style
-~~~~~~~~~~~~~~~~
+* File names should be lowercase with underscores or dashes and end with ``.cpp`` or ``.hpp``.
+* Namespaces should be lowercase with underscores.
+* Class names should be PascalCase. i.e. :code:`AdjArrayBQM`.
+* Type aliases may follow other naming coventions to be more like the standard library. i.e. :code:`MyVector::value_type`
+* Function names should be lowercase with underscores. i.e. :code:`num_variables()`.
+* Variable names should be lowercase with underscores. Private data members should have a trailing underscore.
+* Global variable names should be ``ALL_CAPS_WITH_UNDERSCORES``.
+* Macros should be ``ALL_CAPS_WITH_UNDERSCORES``.
 
-Favor the use of the optional braces for single-line control statements, which enhance consistency and extensibility.
+Format
+~~~~~~
 
-Example:
+* When starting a new C++ project, use clang-format with the .clang-format file included here.
```

Comment:
Nice to have:  "... with the `.clang-format <.clang-format>`_ file included ..."